rotkehlchen/tests/data/mocks/test_coingecko/coins/ethereum.json

```diff
@@ -0,0 +1 @@
+{"symbol":"eth","name":"Ethereum","description":{"en":"Ethereum is a global, open-source platform for decentralized applications. In other words, the vision is to create a world computer that anyone can build applications in a decentralized manner; while all states and data are distributed and publicly accessible. Ethereum supports smart contracts in which developers can write code in order to program digital value. Examples of decentralized apps (dapps) that are built on Ethereum includes tokens, non-fungible tokens, decentralized finance apps, lending protocol, decentralized exchanges, and much more.\r\n\r\nOn Ethereum, all transactions and smart contract executions require a small fee to be paid. This fee is called Gas. In technical terms, Gas refers to the unit of measure on the amount of computational effort required to execute an operation or a smart contract. The more complex the execution operation is, the more gas is required to fulfill that operation. Gas fees are paid entirely in Ether (ETH), which is the native coin of the blockchain. The price of gas can fluctuate from time to time depending on the network demand."},"image":{"thumb":"https://assets.coingecko.com/coins/images/279/thumb/ethereum.png?1595348880","small":"https://assets.coingecko.com/coins/images/279/small/ethereum.png?1595348880","large":"https://assets.coingecko.com/coins/images/279/large/ethereum.png?1595348880"}}
```

Comment:
Hmmm interesting. I see we parse the description and save it in memory in the `CoingeckoAssetData` struct. And unless I am totally mistaken this is not used anywhere except from here: https://github.com/rotki/rotki/blob/d01f565e7066bca1c216c6ebbadef2d231946345/rotkehlchen/icons.py#L87

And here we only parse the image url. Correct? So can we get read of the asset description? If yes, please:

1. Do it. This will save us a lot of memory.
2. Also remove it from these mocks. Both this and the other file